### PR TITLE
adding new resource types to the usage summary struct

### PR DESCRIPTION
**Pull request by RoxyFarhad**
We are splitting binary+tabular data outside of cloud storage and data_upload/egress into binary+tabular qualifiers. This is the change to show that new usage split on the usage panel on the billing admin.

### CODE REVIEW

File: proto/viam/app/v1/billing.proto [hunk 1]
```diff
@@ -32,9 +32,15 @@ message CurrentMonthUsageSummary {
...
-  double total_usage_without_discount = 11; // returns amt without any discounts applied
+  double binary_cloud_storage_usage = 9;
+  double tabular_cloud_storage_usage = 10;
+  double tabular_data_egress = 11; 
+  double tabular_data_upload = 12;
+  double binary_data_egress = 13;
+  double binary_data_upload = 14;
+  double total_usage_quantity = 15;
+  double total_usage_with_discount = 16; // returns amt with any discounts applied
+  double total_usage_without_discount = 17; // returns amt without any discounts applied
```
Comment (RoxyFarhad) [inline proto/viam/app/v1/billing.proto:43]: n.b. we can remove the old types once the app change is merged to show the split and the old types are no longer being used.

**Comment by RoxyFarhad** [comment]
@agreenb I added the new resource types for the data team too - can you just check that the types there are correct? Thanks!

File: proto/viam/app/v1/billing.proto [hunk 2]
```diff
@@ -108,6 +108,12 @@ message GetCurrentMonthUsageResponse {
   double data_egres_usage_cost = 5;
   double remote_control_usage_cost = 6;
   double standard_compute_usage_cost = 7;
+  double binary_cloud_storage_usage = 11;
```
Comment (RoxyFarhad) [inline proto/viam/app/v1/billing.proto:111]: the numbers are weird but I wanted to keep the usage order sequential for clarity

**Review by zaporter-work — APPROVED** [review]
[opt] the usage-related members of this message were ordered alphabetically. Up to you if you want to keep it that way.

**Pull request closed without merging**